changelog.md

```diff
@@ -1,3 +1,111 @@
+<a name="5.3.0"></a>
+# 5.3.0 (2019-09-13)
+
+Expected DevTools release in Chrome 79 ([December 2019](https://chromestatus.com/features/schedule))
+
+## New contributors!
+Thanks to our new contributors 👽🐷🐰🐯🐻! 
+Aaron Labiaga <alabiaga@google.com>
+Brian Olore <brian@olore.net>
+E <psybuzz@users.noreply.github.com>
+luiz tanure <letanure@gmail.com>
+mfriesenhahn <mfriesenhahn@google.com>
+
+## Core
+
+* audits: Point best practices audit links to web.dev ([#9538](https://github.com/googlechrome/lighthouse/pull/9538))
+* audits: point PWA audit description links to web.dev ([#9539](https://github.com/googlechrome/lighthouse/pull/9539))
+* audits: Point a11y audit doc links to web.dev ([#9084](https://github.com/googlechrome/lighthouse/pull/9084))
+* audits: Point SEO audit links to web.dev ([#9537](https://github.com/googlechrome/lighthouse/pull/9537))
```

Comment:
four above here could probably be put into one (and maybe moved to the `Docs` section instead of `Core`?).

`point audit descriptions to new web.dev Lighthouse docs ([#9538](https://github.com/googlechrome/lighthouse/pull/9538), [#9539](https://github.com/googlechrome/lighthouse/pull/9539), [#9084](https://github.com/googlechrome/lighthouse/pull/9084), [#9537](https://github.com/googlechrome/lighthouse/pull/9537))`